### PR TITLE
Docs: Create `CODEOWNERS` document for tracking world maintainers.

### PR DESCRIPTION
**Pull request by ThePhar**
## What is this fixing or adding?
As it says on the tin. Unforunately, GitHub doesn't allow automatic PR review requests for users without write access, but this is still useful for keeping track of GitHub usernames of world maintainers. Discord usernames can only go so far, if I need to ping someone in GitHub and I don't know their username.

Also updated `world maintainer.md` document to reference this doc.

## How was this tested?
I read it.

## If this makes graphical changes, please attach screenshots.
N/A

### CODE REVIEW

**Comment by ThePhar** [comment]
image

This is going to contain errors because not every world maintainer has write access (obviously by design), but the document is still helpful and if GitHub ever decides to allow folks with read-only access to be referenced, then we are good for that day.

File: docs/CODEOWNERS
Comment (t3hf1gm3nt) [inline docs/CODEOWNERS:21]: I know Damocles is listed as a maintainer on our list on discord but unsure what their github username would be, if they have one, as they don't show up in the commit history for the Blasphemous world folder

File: docs/CODEOWNERS
Comment (t3hf1gm3nt) [inline docs/CODEOWNERS:57]: Ludwig is also listed as a maintainer on the discord, but again unsure if they have a github account since i dont see them in the commit history

File: docs/CODEOWNERS
Comment (t3hf1gm3nt) [inline docs/CODEOWNERS:45]: have we determined yet if dewin is still considered the maintainer, or are we taking this opportunity to list HK as #Unmaintained ?

File: docs/CODEOWNERS
Comment (t3hf1gm3nt) [inline docs/CODEOWNERS:135]: @blastron should be added

**Review by t3hf1gm3nt — CHANGES_REQUESTED** [review]
So the big thing I have issue with is the ordering. I can see you ordered it by the order they appear in the /worlds/ folder. I feel, however, it should be ordered as it appears on the website's supported games/setup guides pages. I will update my list on the discord accordingly to whichever order we go with. Just would like to have consistency between all the lists.

File: docs/CODEOWNERS
Comment (ThePhar) [reply docs/CODEOWNERS:45]: Until we run through the new "maintainer" vote, I'm still listing him.

File: docs/CODEOWNERS
Comment (ThePhar) [reply docs/CODEOWNERS:21]: Need their github username. :P

File: docs/CODEOWNERS
Comment (ThePhar) [reply docs/CODEOWNERS:57]: Need their github username. :P

File: docs/CODEOWNERS
Comment (ThePhar) [reply docs/CODEOWNERS:135]: Added.

File: docs/CODEOWNERS
Comment (ThePhar) [reply docs/CODEOWNERS]: Done.

File: docs/CODEOWNERS
Comment (ThePhar) [reply docs/CODEOWNERS]: Done.

File: docs/world maintainer.md
Comment (Exempt-Medic) [inline docs/world maintainer.md:46]: "voted" isn't usually used in this way. "gets approved" might be a better wording.

File: docs/CODEOWNERS
Comment (Exempt-Medic) [inline docs/CODEOWNERS:3]: "file" could be changed to "document" to be more formal and match later instances. "notate" is usually used with music or shorthand writing, changing it to "note" or "list" would be better. You can also remove "who" from the sentence as it makes it grammatically incorrect. "content of these worlds" should be "the contents of these worlds"

**Review by Exempt-Medic — CHANGES_REQUESTED** [review]
Grammar improvements/corrections

File: docs/world maintainer.md
Comment (Exempt-Medic) [reply docs/world maintainer.md:46]: Could do "gets voted in", but that also implies top-down, as would most votes unless the wording is something like "gets voted in by the community". Actually "gets voted in by the X" should work well, since it'll also say who's actually doing the voting.

File: docs/CODEOWNERS
Comment (agilbert1412) [inline docs/CODEOWNERS:121]: In a context where we give credit, I'd add Jouramie as a very significant contributor.
But in this context, where the goal is to have someone to contact about the world, I would set myself up as the only name for Stardew Valley (for now). Jouramie has not been actively contributing to the project since version 2.x.x and it is unclear if he plans on contributing again.

**Review by agilbert1412 — APPROVED** [review]

File: docs/CODEOWNERS
Comment (ThePhar) [reply docs/CODEOWNERS:121]: Sure, if that changes let me know and I can update accordingly.

File: docs/CODEOWNERS
Comment (TRPG0) [inline docs/CODEOWNERS:22]: Damocles just does client side stuff for Blasphemous so this should just be me actually.

**Review by TRPG0 — APPROVED** [review]

File: docs/CODEOWNERS
Comment (ThePhar) [reply docs/CODEOWNERS:61]: I don't see an issue with it. If that's what you prefer, I can do that.

File: docs/CODEOWNERS
Comment (ThePhar) [reply docs/CODEOWNERS:56]: The world isn't disabled.

**Review by black-sliver — APPROVED** [review]
Approving changes to world maintainer.md

As for CODEOWNERS, it lgtm,  but maybe we want a second (or third) opinion if that is the way we want to implement it? Since most auto-pings are broken, we could also rename the file and get rid of that "... contains errors" and implement an action to ping down the road.

**Comment by ThePhar** [comment]
I personally don't see it as that much of an issue, plus it still allows auto pings for the few of us that are members. But I'll wait til we start RC phase for any additional opinions before I merge in.

**Comment by BadMagic100** [comment]
I have spent some time looking into actions workflows which can do auto-pings/auto-review-request based on CODEOWNERS; some seem to exist. But for now I don't personally believe it to be a blocker to merging this, nor do I believe renaming the file or the "errors" it avoids really matters

**Pull request merged**
